### PR TITLE
Add !!mixinextras tag

### PR DESCRIPTION
**Pull request by Awakened-Redstone**

### CODE REVIEW

File: tags/lib/mixinextras.ytag [hunk 1]
```diff
@@ -0,0 +1,9 @@
+type: text
+
+---
+
+Mixin Extras is a small companion library for the Mixin system, offering better ways to use Mixins.
```
Comment (Technici4n) [inline tags/lib/mixinextras.ytag:5]: Not necessarily better, just more options. It won't help you if all you need is a simple `@Inject`.

File: tags/lib/mixinextras.ytag [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+It is highly recommended to use Mixin Extras as it offers features such as `ModifyExpressionValue` which is a better alternative to `Redirect`
```
Comment (Technici4n) [inline tags/lib/mixinextras.ytag:6]: Once again, not necessarily a better alternative, but definitely a more compatible alternative depending on the use case.

**Review by Technici4n — CHANGES_REQUESTED** [review]

File: tags/lib/mixinextras.ytag (hunk 1 above)
Comment (Awakened-Redstone) [reply tags/lib/mixinextras.ytag]: Replaced "better" with "alternative"

File: tags/lib/mixinextras.ytag (hunk 2 above)
Comment (Awakened-Redstone) [reply tags/lib/mixinextras.ytag]: Rephrased it to recommend it over "highly recommending" and avoid calling the features as "better"

**Review by Technici4n — APPROVED** [review]

**Comment by Awakened-Redstone** [comment]
Oh crap! I forgot to put the link between `<>`

**Comment by LlamaLad7** [comment]
`@WrapOperation` is more the "chainable alternative to `Redirect`", though I feel like mentioning that concept at all might lead to more people replacing every Redirect with it and making things worse. Might at least be worth a clarifying sentence to explain that it's only better if you're not completely replacing the original operation.

**Pull request merged**